coordinape/coordinape
# fixes #435 and fixes #439 by porting uploadProfileAvatar and uploadProfileBackground from laravel to hasura

This includes porting the FE as well. There is some unfinished business in here w/ FE patterns but it is functional for now and I left good comments for future improvements. 

fixes #435 

Test plan after deploy to staging/production:

1. visit My Profile by clicking your avatar in top right
2. click Edit profile
3. click the pencil icon on avatar
4. choose an avatar image
5. click the little upload button that shows up on the avatar
5. make sure it finishes ok
6. right click on the avatar now and copy the image url, open it in a new tab , should be an s3 image url
7. change the avatar again (steps 3-5)
8. reload the other tab, that image should be gone/deleted now

Perform a similar test for edit background, it's on the main profile page and has same workflow. 


*FOR DEPLOYMENT* 
These api keys need to be set or the instance that is running vercel needs to have the proper EC2 Role, I'm not sure how that works. 
```
IMAGES_AWS_ACCESS_KEY_ID=tbd
IMAGES_AWS_SECRET_ACCESS_KEY=tbd
```
